src/mountpoint.h

```diff
@@ -0,0 +1,37 @@
+#ifndef SRC_MOUNTPOINT_H_
+#define SRC_MOUNTPOINT_H_
+
+/*
+ * Copyright 2017 resin.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <windows.h>
```

Comment:
This file should not rely on windows specific headers (or be moved into `src/windows`) – it doesn't look like they're required here anyways – copy / paste accident? :D